regulations/tasks.py

```diff
@@ -24,55 +25,70 @@
 
 from django.conf import settings
 from django.template import loader
+from django.utils.crypto import get_random_string
 
 from regulations.models import FailedCommentSubmission
 logger = get_task_logger(__name__)
 
 
 @shared_task(bind=True)
-def submit_comment(self, body):
+def submit_comment(self, comments, form, meta):
     '''
     Submit the comment to regulations.gov. If unsuccessful, retry the task.
     Number of retries and time between retries is managed by Celery settings.
     The main comment is converted to a PDF and added as an attachment; the
     'general_comment' field refers to this attachment.
 
-    :param body: dict of fields and values to be posted to regulations.gov
+    :param comments: List of sectional comments
+    :param form: Dict of form data
```

Comment:
Would it be better to replace "Dict of form data" with "Additional comment fields accepted by regulations.gov"?